### PR TITLE
[UI]:Workspace Page MUI migration from v4 to Sistent components 

### PR DESCRIPTION
**Pull request by SAHU-01**
**Notes for Reviewers**

This PR fixes part of  #11493 , based on [this](https://docs.google.com/spreadsheets/d/1RlUcr-iLtCFCsV1VZvWD0jBijPJAx-Rk-1GLLsL9sSI/edit?usp=sharing) sheet , the workspace page and its related components have been migrated to use sistent components and necessary mui v5 changes have been applied to these pages.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by SAHU-01** [comment]
> > @sudhanshutech the `SearchBar` used in the `index.js` file [_import SearchBar from '../../../utils/custom-search'_] is not the sistent `SearchBar` , would you recommend using the sistent `SearchBar` here, if yes, I'll implement it.
> 
> where this search bar from utils being used? Because we already use searc bar from sistent.

Here the utils SearchBar is being used:
image

**Comment by SAHU-01** [comment]
> use it from sistent directly . Search bar is already in sistent . See other places for example , it might have missed .

Got it, yes I noticed other places using the sistent, so I was wondering why we had this utils one here, either ways, I'll implement the sistent one here.

**Review by Ajay-singh1 — CHANGES_REQUESTED** [review]
We need to import the components from @material/mui/core not from @material-ui/core.

**Comment by leecalcote** [comment]
> We need to import the components from @mui/material/core not from @material-ui/core.

Thanks for helping out here, @Ajay-singh1 👍

**Comment by sudhanshutech** [comment]
@SAHU-01 any caveats here? or everything is fine?

**Comment by SAHU-01** [comment]
@sudhanshutech I've tested all the features for workspace and there isn't any sort of glitch or potential issues I think this code would cause, I'll probably add a demo video of how this works locally testing all features.

File: ui/components/Lifecycle/Workspaces/index.js [hunk 1]
```diff
@@ -44,19 +46,50 @@ import PromptComponent, { PROMPT_VARIANTS } from '../../PromptComponent';
...
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  whiteSpace: 'nowrap',
+});
+
+export const ToolWrapper = styled('div')(({ theme }) => ({
+  marginBottom: '3rem',
+  display: 'flex',
+  justifyContent: 'space-between',
+  backgroundColor: theme.palette.mode === 'dark' ? '#464646' : '#FFFFFF',
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Workspaces/index.js:66]: we should use sistent tokens instead hard code hex colors

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Review by aabidsofi19 — APPROVED** [review]

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**